app/javascript/redux/configureStore.js

```diff
@@ -0,0 +1,29 @@
+import { applyMiddleware, createStore } from "redux";
+
+import thunk from "redux-thunk";
+// import { composeWithDevTools } from 'redux-devtools-extension';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.